worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -991,7 +995,7 @@ class PushDataHandler extends BaseMessageHandler with Logging {
 
     val (messageMaster, messageSlave) =
       messageType match {
-        case Type.PUSH_DATA | Type.PUSH_DATA_HAND_SHAKE =>
```

Comment:
duplicate match

https://github.com/apache/incubator-celeborn/blob/7d7107d607b853ab61b8299f83de2e3dacfa7e3c/worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala#L994-L1000

Reply:
so this is a bug?

Reply:
> PushDataHandler.scala:1003: unreachable code

> so this is a bug?

I think so.
The status code is just passed to `CelebornIOException`, so probably doesn't affect anything.

Reply:
better to make separated PR for such cases.

Reply:
OK